code/common/html.q

```diff
@@ -67,7 +67,7 @@ jsformat:{ k:cols x; flip k !(y value t:type each x)@'value x:flip 0!x}
 // Arg: dictionary decoded front end JSON  
 // format should be `func`arg1`arg2 ... `arg8!(function;arg1;arg2;...;arg3)
 // all args except arg1 are optional
-execdict:{$[not `func in key x;'"no func in dictionary";1=count key x;(value x`func) @ 1;1<count key x;(value x`func) . value x _ `func]}
+execdict:{$[not `func in key x;'"no func in dictionary";1=count key x;(value x`func) @ 1;1<count key x;(value x`func) . value x _ `func;]}
```

Comment:
``value[x`func]`` should work. You can probably remove `@` here too and index without it.

Reply:
What might be even better is have:
```
if[not`func in key x;...];
:value[x`func].$[1=count key x;(),1;value x _`func];
```
Untested but I think it should work.